wagtail_modeltranslation/static/wagtail_modeltranslation/js/version_compare.js

```diff
@@ -8,20 +8,20 @@
  * @param {object} [options] Optional flags that affect comparison behavior:
  * <ul>
  *     <li>
- *         <tt>lexicographical: true</tt> compares each part of the version strings lexicographically instead of
+ *         <tt>lexicographical: false</tt> compares each part of the version strings lexicographically instead of
  *         naturally; this allows suffixes such as "b" or "dev" but will cause "1.10" to be considered smaller than
  *         "1.2".
  *     </li>
  *     <li>
- *         <tt>zeroExtend: true</tt> changes the result if one version string has less parts than the other. In
+ *         <tt>zeroExtend: false</tt> changes the result if one version string has less parts than the other. In
  *         this case the shorter string will be padded with "zero" parts instead of being considered smaller.
```

Comment:
Can you explain this?

Reply:
Sorry, this was wrong. I mixed it up with the default values of the parameter. I corrected and clarified the js doc in the newest commit.

Actually I'm unsure, whether this comparison should happen in js at all. Probably one should create different js files for different Wagtail versions and do the switching in `wagtail_hooks.py`. What do you think?